src/composables/queries/useUserClaimsQuery.ts

```diff
@@ -74,7 +74,6 @@ export default function useUserClaimsQuery(
   const queryOptions = reactive({
     enabled: isQueryEnabled,
     refetchOnMount: false,
-    refetchOnWindowFocus: false,
```

Comment:
I found that without removing this the pending rewards pill didn't update on switching networks.